### PR TITLE
iftop: update to 2018-10-03

### PR DESCRIPTION
**Pull request by heitbaum**
iftop: update to 2018-10-03
- update 1.0pre4 (2014-01-19) to 77901c8c (2018-10-03)
- changelog: https://code.blinkace.com/pdw/iftop/commits/master
- fix compile with libpcap-1.10.0
- updated PKG_URL and PKG_SITE

network-tools: fix iftop compile failure (108)

### CODE REVIEW

**Comment by heitbaum** [comment]
Fixes introduced failure from #4953

File: packages/addons/tools/network-tools/changelog.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@
 - bwm-ng: update to 0.6.3
 - tcpdump: update to 4.99.0
 - libpcap: revert to building with configure (107)
+- iftop: update to 2018-10-03 and fix compile with libpcap-1.10.0
```
Comment (CvH) [inline packages/addons/tools/network-tools/changelog.txt:5]:
```suggestion
- iftop: update to 2018-10-03
```

and pls sort changelog abc for that entry

File: packages/addons/addon-depends/network-tools-depends/iftop/package.mk [hunk 2]
```diff
@@ -2,11 +2,11 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="iftop"
-PKG_VERSION="1.0pre4"
-PKG_SHA256="f733eeea371a7577f8fe353d86dd88d16f5b2a2e702bd96f5ffb2c197d9b4f97"
+PKG_VERSION="77901c8c53e01359d83b8090aacfe62214658183"
```
Comment (CvH) [inline packages/addons/addon-depends/network-tools-depends/iftop/package.mk:5]: is the latest git version needed to make it work?

Comment (heitbaum) [reply packages/addons/addon-depends/network-tools-depends/iftop/package.mk:5]: No. I did review Paul’s changes and believe they have merit - eg 1Gbps/DNS

- I have reached out to Paul with our patches too.
- I did consider (and look at incorporating some of his patches) but the resulting patch was huge ...

File: packages/addons/tools/network-tools/changelog.txt (hunk 1 above)
Comment (heitbaum) [reply packages/addons/tools/network-tools/changelog.txt]: Done. Fixed

**Review by CvH — APPROVED** [review]

**Pull request merged**